<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>817</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1136</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>26</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:qlineargradient(x1:0, y1:1, x2:1, y2:0, stop:0 #D900D9, stop:1 #00E5E5)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
     <item>
      <widget class="QLabel" name="original_image">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>500</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="res_image">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>500</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #de2aa2;
color: white;
</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generate_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #de2aa2;
color: white;
</string>
         </property>
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="browse_image_button">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(94, 92, 100);
background-color: rgb(220, 138, 221);
font-size:14px;</string>
         </property>
         <property name="text">
          <string>Browse Image</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="image_description">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: white;
color: black;
gridline-color: rgb(186, 205, 168);
font-size:14px;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Enter image's URL</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="browse_emoji_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(94, 92, 100);
background-color: rgb(220, 138, 221);
font-size:14px;
</string>
         </property>
         <property name="text">
          <string>Browse Emoji</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="emoji_description">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: white;
color: black;
gridline-color: rgb(186, 205, 168);
font-size:14px;</string>
         </property>
         <property name="placeholderText">
          <string>Enter emoji's URL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
